containers/java/.devcontainer/base.Dockerfile

```diff
@@ -26,18 +28,26 @@ ARG INSTALL_GRADLE="false"
 ARG GRADLE_VERSION=""
 ENV SDKMAN_DIR="/usr/local/sdkman"
 ENV PATH="${SDKMAN_DIR}/candidates/java/current/bin:${PATH}:${SDKMAN_DIR}/candidates/maven/current/bin:${SDKMAN_DIR}/candidates/gradle/current/bin"
-RUN bash /tmp/library-scripts/java-debian.sh "none" "${SDKMAN_DIR}" "${USERNAME}" "true" \
-    && if [ "${INSTALL_MAVEN}" = "true" ]; then bash /tmp/library-scripts/maven-debian.sh "${MAVEN_VERSION:-latest}" "${SDKMAN_DIR}" ${USERNAME} "true"; fi \
-    && if [ "${INSTALL_GRADLE}" = "true" ]; then bash /tmp/library-scripts/gradle-debian.sh "${GRADLE_VERSION:-latest}" "${SDKMAN_DIR}" ${USERNAME} "true"; fi \
-    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+# Install Java tools such as JDK, Maven and Gradle
+RUN bash /tmp/library-scripts/java-debian.sh $TARGET_JAVA_VERSION "${SDKMAN_DIR}" "${USERNAME}" "true" \
+	&& if [ "${INSTALL_MAVEN}" = "true" ]; then bash /tmp/library-scripts/maven-debian.sh "${MAVEN_VERSION:-latest}" "${SDKMAN_DIR}" ${USERNAME} "true"; fi \
+	&& if [ "${INSTALL_GRADLE}" = "true" ]; then bash /tmp/library-scripts/gradle-debian.sh "${GRADLE_VERSION:-latest}" "${SDKMAN_DIR}" ${USERNAME} "true"; fi \
+	&& apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+# Add symlinks to the installed JDK
+RUN ls ${SDKMAN_DIR}/candidates/java | grep "^${TARGET_JAVA_VERSION}.*" > /tmp/library-scripts/jdkname \
```

Comment:
Wouldn't be better to use the image mcr image as suggested by [ms docs](https://docs.microsoft.com/en-us/java/openjdk/containers#use-a-different-base-os-image) ?

Reply:
I do recommend to use the `COPY --from=...` instruction instead of SDKMan in this case because we do not manage SDKMan manifest, and there is likely to be a time gap between our release and whenever SDKMan updates the manifest.